### ISSUE TITLE
External Blocklists

### ISSUE THREAD

**Issue by CarolDas** [bookwyrm-social/bookwyrm#3413]
Is it possible to integrate the [FediBlackBook API](https://fediblackbook.com/) with BookWyrm?

**Comment by mouse-reeve** [comment]
Definitely! We currently only have a way to add blocklists at the instance level, so we'd need to add this as a new tool in the admin panel.
